instance_core_stats.tf

```diff
@@ -10,15 +10,26 @@ resource "openstack_compute_instance_v2" "grafana" {
   }
 }
 
-resource "openstack_blockstorage_volume_v2" "grafana-data" {
-  name        = "stats"
-  description = "Data volume for Grafana"
-  size        = 2
+# 17.4.2024: This resource creation is commented out because the volume
+# from the old cloud is being attached to the instance in the new cloud.
+# Found 2 volumes attached to the instance in the old cloud. So, adding both
+# of them here and creating the volume attachment resources below for both.
+# resource "openstack_blockstorage_volume_v2" "grafana-data" {
+#   name        = "stats"
+#   description = "Data volume for Grafana"
+#   size        = 2
+# }
+
+resource "openstack_compute_volume_attach_v2" "stats-data-vol1" {
+  instance_id = openstack_compute_instance_v2.grafana.id
+  volume_id   = "ad56f01f-cb21-458c-b1f9-90f937aa6525"
+  device      = "/dev/vdb"
 }
 
-resource "openstack_compute_volume_attach_v2" "stats-va" {
+resource "openstack_compute_volume_attach_v2" "stats-data-vol2" {
   instance_id = openstack_compute_instance_v2.grafana.id
-  volume_id   = openstack_blockstorage_volume_v2.grafana-data.id
+  volume_id   = "254b499b-1293-4341-ba52-2901bbe56fcb"
+  device      = "/dev/vdc"
```

Comment:
Both volumes can be removed because I don't think either of them is being used. The dashboards are being provisioned from a local directory `/data/monitoring/grafana_data`. I think we can leave this for now. I will create a Grafana-dedicated issue (includes, grafana update, dashboard automation (both import and export), NFS directory for data, cleanup of existing data/unsued mounts/etc. tasks) soon.